addons/l10n_ae/views/report_invoice_templates.xml

```diff
@@ -104,14 +104,6 @@
             </td>
         </xpath>
 
-        <xpath expr="//td[@name='td_subtotal']/span" position="attributes">
-            <attribute name="t-if">o.company_id.country_id.code != 'AE' or o.company_price_include == 'tax_excluded'</attribute>
-        </xpath>
-        
-        <xpath expr="//td[@name='td_subtotal']/span" position="after">
-            <span t-if="o.company_id.country_id.code == 'AE' and o.company_price_include == 'tax_included'" class="text-nowrap" t-field="line.price_total">31.05</span>
```

Comment:
You could add class `d-none` to make it invisible

Reply:
another possibility could be to just change the t-if to t-if="0", it depends if we think the t-if has higher probability to be targeted than `class` or not